.github/workflows/linux_riscv.yml

```diff
@@ -0,0 +1,188 @@
+name: Linux RISC-V with Conan (Ubuntu 22.04, Python 3.10)
+on:
+  schedule:
+    # at 00:00 on Wednesday and Saturday
+    - cron: '0 0 * * 3,6'
+  workflow_dispatch:
+  pull_request:
+    paths-ignore:
+      - '**/docs/**'
+      - 'docs/**'
+      - '**/**.md'
+      - '**.md'
+      - '**/layer_tests_summary/**'
+      - '**/conformance/**'
+  push:
+    paths-ignore:
+      - '**/docs/**'
+      - 'docs/**'
+      - '**/**.md'
+      - '**.md'
+      - '**/layer_tests_summary/**'
+      - '**/conformance/**'
+    branches:
+      - master
+      - 'releases/**'
+
+concurrency:
+  # github.ref is not unique in post-commit
+  group: ${{ github.event_name == 'push' && github.run_id || github.ref }}-linux-riscv
+  cancel-in-progress: true
+
+jobs:
+  Build:
+    defaults:
+      run:
+        shell: bash
+    runs-on: aks-linux-16-cores
+    container:
+      image: ubuntu:22.04
```

Comment:
why not the internal one, `openvinogithubactions.azurecr.io/dockerhub/ubuntu:20.04`? Because there is a need for `22.04`? There might be a problem with Docker Hub rate limits. However, `ubuntu:22.04` should be cached as a base image and not pulled every time.

Reply:
yes, I need Ubuntu 22.04 because it has support for RISCV
Good point to have `openvinogithubactions.azurecr.io/dockerhub/ubuntu:22.04`, I will ask Andrey about it

Reply:
fixed here https://github.com/openvinotoolkit/openvino/pull/20078